http/src/main/resources/reference.conf

```diff
@@ -153,6 +153,9 @@ proxy {
   networkTag = "leonardo"
   projectVPCNetworkLabel = "vpc-network-name"
   projectVPCSubnetLabel = "vpc-subnetwork-name"
+  # Subnets are regional and independent of the runtime region. Networks on the other hand are global.
+  # See https://cloud.google.com/vpc/docs/vpc
+  projectVPCSubnetRegion = "us-central1"
```

Comment:
does this region apply to all subnet configs?

Reply:
Yeah it's globally set in Leo (as is the Dataproc/GCE region). If/when we support other regions we might want to start choosing this dynamically.